org.json.JSONObject Dependency injection Linkage error for Keycloak enabled environments

<!--
If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (accessible in the right sidebar) for an explanation about the information we'd like you to provide.
-->
### Bug Description

After the introduction of the `com.jayway.jsonpath` dependency causes the KeycloakService to crash when loading the `net.minidev.json` library because it tries to load org.json.JSONObject and results in a conflict.

```
Caused by: java.lang.LinkageError: loader constraint violation: loader org.glassfish.javaee.full.deployment.EarClassLoader @6bb6c608 wants to load class net.minidev.json.JSONObject. A different class with the same name was previously loaded by org.apache.felix.framework.BundleWiringImpl$BundleClassLoader
```

### Steps to Reproduce
1. Start the latest 1.51.0-SNAPSHOT version in a dockerized environment with Keycloak enabled as a Authorization Provider
2. Create a new user with the admin


### Additional Information
Suggested solution: 
Since KeycloakService is the only place where `net.minidev.json` is used, replace it with JSONPath since it also simplifies the code there.

--- a/sormas-backend/src/main/java/de/symeda/sormas/backend/user/KeycloakService.java
+++ b/sormas-backend/src/main/java/de/symeda/sormas/backend/user/KeycloakService.java
@@ -21,7 +21,6 @@ package de.symeda.sormas.backend.user;
 import static java.util.Collections.singletonList;
 import static org.keycloak.representations.IDToken.LOCALE;
 
-import java.text.ParseException;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
@@ -39,6 +38,7 @@ import javax.enterprise.event.Observes;
 import javax.ws.rs.WebApplicationException;
 import javax.ws.rs.core.Response;
 
+import com.jayway.jsonpath.JsonPath;
 import org.apache.commons.collections.CollectionUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.eclipse.microprofile.config.ConfigProvider;
@@ -51,8 +51,6 @@ import org.keycloak.representations.idm.UserRepresentation;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.nimbusds.jose.util.JSONObjectUtils;
-
 import de.symeda.sormas.api.AuthProvider;
 import de.symeda.sormas.api.Language;
 import de.symeda.sormas.api.user.UserRole;
@@ -61,7 +59,6 @@ import de.symeda.sormas.backend.user.event.MockUserCreateEvent;
 import de.symeda.sormas.backend.user.event.PasswordResetEvent;
 import de.symeda.sormas.backend.user.event.UserCreateEvent;
 import de.symeda.sormas.backend.user.event.UserUpdateEvent;
-import net.minidev.json.JSONObject;
 
 /**
  * @author Alex Vidrean
@@ -78,8 +75,7 @@ public class KeycloakService {
 
 	private static final String OIDC_REALM = "realm";
 	private static final String OIDC_SERVER_URL = "auth-server-url";
-	private static final String OIDC_CREDENTIALS = "credentials";
-	private static final String OIDC_SECRET = "secret";
+	private static final String OIDC_SECRET = "credentials.secret";
 
 	private static final String REALM_NAME = "SORMAS";
 
@@ -106,20 +102,16 @@ public class KeycloakService {
 			return;
 		}
 
-		try {
-			JSONObject json = JSONObjectUtils.parse(oidcJson.get());
+		String keycloakJsonConfig = oidcJson.get();
 
-			keycloak = KeycloakBuilder.builder()
-				.realm(json.getAsString(OIDC_REALM))
-				.serverUrl(json.getAsString(OIDC_SERVER_URL))
-				.clientId("sormas-backend")
-				.clientSecret(JSONObjectUtils.getJSONObject(json, OIDC_CREDENTIALS).getAsString(OIDC_SECRET))
-				.grantType(OAuth2Constants.CLIENT_CREDENTIALS)
-				.build();
+		keycloak = KeycloakBuilder.builder()
+			.realm(JsonPath.read(keycloakJsonConfig, OIDC_REALM))
+			.serverUrl(JsonPath.read(keycloakJsonConfig, OIDC_SERVER_URL))
+			.clientId("sormas-backend")
+			.clientSecret(JsonPath.read(keycloakJsonConfig, OIDC_SECRET))
+			.grantType(OAuth2Constants.CLIENT_CREDENTIALS)
+			.build();
 
-		} catch (ParseException e) {
-			throw new IllegalArgumentException("Invalid JSON for backend keycloak oidc");
-		}
 	}
 
 	public void handleUserCreateEvent(@Observes UserCreateEvent userCreateEvent) {

